### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by HikaruIchijyo** [komikku-app/komikku#202] [General]
isn't it possible to use j2k ui or build the komikku based on it & then migrate stuff to new framework & add other features.

just curious.
cuz that ui is more friendly 
one handed etc

**Comment by cuong-tran** [comment]
J2K is based on old UI framework, the new framework is better.
 Also, this allows migrating new features/changes from the main stream Mihon.

That said, I also think the J2K's UI is nice and would implement it sometime later as an option.

**Reply by HikaruIchijyo** [reply]
> J2K is based on old UI framework, the new framework is better.
>  Also, this allows migrating new features/changes from the main stream Mihon.
> 
> That said, I also think the J2K's UI is nice and would implement it sometime later as an option.

ik 
but somehow j2k ui is fast & responsive

one handed ui is good 

maybe start implementing those ui features one by one since most of the forks of j2k are not soo good

**Comment by Abinpj1** [comment]
Can someone please tell me how I can add animes? Mines just asked empty, where do I find the url repos for this

**Reply by kana-shii** [reply]
We do not endorse repos, so we can't provide them. You can watch anime using Anikku.

**Reply by cuong-tran** [reply]
Ask any AI: ChatGPT, Gemini, Copilot... You will get your answer & guide for that.
